src/components/Incorporation/IncorporationDateTime.vue

```diff
@@ -120,28 +120,30 @@ export default class IncorporationDateTime extends Mixins(DateMixin) {
   @Prop({ default: null }) readonly effectiveDateTime!: EffectiveDateTimeIF
 
   // Local properties
-  private isImmediate: boolean = false
-  private isFutureEffective: boolean = false
+  protected isImmediate = false
+  protected isFutureEffective = false
 
   // Date properties
-  private selectDate: string = ''
-  private dateText: string = ''
-  private datePicker: string = ''
+  protected selectDate = ''
+  protected dateText = ''
+  protected datePicker = ''
 
   // Time properties
-  private selectHour: string[] = []
-  private selectMinute: string[] = []
-  private selectPeriod: string = 'AM'
+  protected selectHour = [] as string[]
+  protected selectMinute = [] as string[]
+  protected selectPeriod = 'AM'
 
   // Date Time Selectors
-  private hours: Array<string> = [...Array(12).keys()]
+  readonly hours: Array<string> = [...Array(12).keys()]
     .map(num => (++num).toLocaleString('en-US'))
```

Comment:
This is a "constant" (but needs to be declared as readonly because it's a class member).

These selectors don't change. One more way of saying is they are static.